package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "systemjs-builder",
-  "version": "0.15.35",
+  "version": "0.15.36",
```

Comment:
This seems to be a PR against the 0.16 branch instead of the 0.15 one? Happy to work with either though.